### PR TITLE
US-623222: Enable HIGH_SECURE_CRYPTO_MODE for SRS

### PR DESCRIPTION
**Pull request by bhowd1**
- Added highSecureCryptoMode flag changes to deployment and helper files
- Updated docs

### CODE REVIEW

**Review by chandra-kalavakuri — APPROVED** [review]

File: charts/backingservices/charts/srs/values.yaml [hunk 1]
```diff
@@ -27,3 +27,7 @@ srsRuntime:
     AuthEnabled: false
     # When `AuthEnabled` is `true`, enter the appropriate public key URL. When `AuthEnabled` is `false`(default), leave this parameter empty.
     OAuthPublicKeyURL: ""
+
+  # Set to `true` if Higly secured connection complying NIST SP 800-53 and NIST SP 800-131 is required; otherwise leave set to `false`
+  highSecureCryptoMode:
```
Comment (bhowd1) [inline charts/backingservices/charts/srs/values.yaml:32]: use `enableSecureCrypto`

**Comment by kingakowalska1** [comment]
Hi Divyansh,

I have some comments from the doc side. 😊 

Could we make the following changes to these sentences in all the places they appear? 

For enableSecureCryptoMode:

Set to `true` if Higly secured connection complying NIST SP 800-53 and NIST SP 800-131 is required; otherwise leave set to `false`..

To 

Set to `true` if you require a highly secured connection that complies with NIST SP 800-53 and NIST SP 800-131. Otherwise set to `false`.

For javaOpts:

This is used to configure Java options values.

To 

Use this parameter to configure values for Java options. 

(Additionally, are there any example values we can provide here?)



And seeing as we’re already making commits to this part, could we also remove the additional period at the end of this sentence for k8sProvider? 

Specify your Kubernetes provider name. Supported values are [`eks`, `aks`, `minikube`, `gke`, `openshift`, `pks`]..

To

Specify your Kubernetes provider name. Supported values are [`eks`, `aks`, `minikube`, `gke`, `openshift`, `pks`].


Thanks!
Kinga

**Comment by bhowd1** [comment]
> Hi Divyansh,
> 
> I have some comments from the doc side. 😊
> 
> Could we make the following changes to these sentences in all the places they appear?
> 
> For enableSecureCryptoMode:
> 
> Set to `true` if Higly secured connection complying NIST SP 800-53 and NIST SP 800-131 is required; otherwise leave set to `false`..
> 
> To
> 
> Set to `true` if you require a highly secured connection that complies with NIST SP 800-53 and NIST SP 800-131. Otherwise set to `false`.
> 
> For javaOpts:
> 
> This is used to configure Java options values.
> 
> To
> 
> Use this parameter to configure values for Java options.
> 
> (Additionally, are there any example values we can provide here?)
> 
> And seeing as we’re already making commits to this part, could we also remove the additional period at the end of this sentence for k8sProvider?
> 
> Specify your Kubernetes provider name. Supported values are [`eks`, `aks`, `minikube`, `gke`, `openshift`, `pks`]..
> 
> To
> 
> Specify your Kubernetes provider name. Supported values are [`eks`, `aks`, `minikube`, `gke`, `openshift`, `pks`].
> 
> Thanks! Kinga

Hi @kingakowalska1, Thanks for your review comments, I've addressed in the latest commit.

**Review by Saurabh-16 — APPROVED** [review]

**Review by kingakowalska1 — APPROVED** [review]
Great work, thanks!

**Pull request merged**